SpotifyAPI/Local/SpotifyLocalAPI.cs

```diff
@@ -149,21 +157,72 @@ public StatusResponse GetStatus()
         }
 
         /// <summary>
-        /// Mutes Spotify (Requires nircmd.dll)
+        /// Mutes Spotify (Requires Windows 7 or newer)
         /// </summary>
         public void Mute()
         {
-            if (File.Exists("nircmd.dll"))
-                DoNirCmd("muteappvolume spotify.exe 1");
+            //Windows < Windows Vista Check
+            Contract.Requires(Environment.OSVersion.Version.Major >= 6);
```

Comment:
Mh, I actually like the idea, but I wonder if it's good to put it in a library. ("if-throw-else" seems to be a more general solution, let me think and research a bit about it before changing :+1: )